js/web/test/data/ops/pad_f16.jsonc

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "name": "constant 2D",
+    "operator": "Pad",
+    "opset": { "domain": "", "version": 10 },
```

Comment:
Can you also add a test for version > 10, in which the value is as an input not attribute and have the type `float16`? I suppose this can test the float16 uniform?

Do you need to update the code [here](https://github.com/microsoft/onnxruntime/blob/main/js/web/lib/wasm/jsep/backend-webgpu.ts#L466-L467)?

Reply:
done